packages/js/package.json

```diff
@@ -14,8 +15,8 @@
     "test": "jest",
     "test:coverage": "jest --silent --env=jsdom && jest --silent --coverage",
     "test:report": "codecov -F js",
-    "prepublish": "npm run build && npm run package",
-    "report": "WITH_REPORT=true npm run package"
+    "prepublish": "pnpm run build && pnpm run package",
```

Comment:
seems original `npm` works fine?

Reply:
yes. Update it due to [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1644474687361579?thread_ts=1644473115.488029&cid=C02A8G4HVAM) on Slack.
I'm good with both (npm and pnpm) here.

Any better idea?

Reply:
I'll go with `pnpm` if you ask me.  ;-)

Reply:
Finally `"prepublish": "pnpm package"`.
See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1644549967093349) on Slack